### PR TITLE
Feature/428 fixes

### PR DESCRIPTION
**Pull request by kkuepper**
- add all search results to the queue (same as BMM app)
- remove duplicate contributors (e.g. track/108815)

### CODE REVIEW

File: utils/uniqueItems.ts
Comment (sifferhans) [inline utils/uniqueItems.ts:1]: Couldn't we use a [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set) here?

**Review by sifferhans — APPROVED** [review]
Just a question about the `uniqueItems` function, other than that looks good

File: utils/uniqueItems.ts
Comment (kkuepper) [reply utils/uniqueItems.ts:1]: I guess. But filter is working as well. So why change it after is has been implemented and tested?

Comment (sifferhans) [reply utils/uniqueItems.ts:1]: Yeah, of course 👍🏼

**Pull request merged**